Framework/Algorithms/src/PDFFourierTransform.cpp

```diff
@@ -229,12 +231,12 @@ double PDFFourierTransform::determineRho0() {
 void PDFFourierTransform::exec() {
   // get input data
   API::MatrixWorkspace_const_sptr inputWS = getProperty("InputWorkspace");
-  MantidVec inputQ = inputWS->dataX(0);                   //  x for input
+  auto inputQ = inputWS->x(0).rawData();                    //  x for input
   HistogramData::HistogramDx inputDQ(inputQ.size(), 0.0); // dx for input
   if (inputWS->sharedDx(0))
     inputDQ = inputWS->dx(0);
-  MantidVec inputFOfQ = inputWS->dataY(0);  //  y for input
-  MantidVec inputDfOfQ = inputWS->dataE(0); // dy for input
+  auto inputFOfQ = inputWS->y(0).rawData();  //  y for input
+  auto inputDfOfQ = inputWS->e(0).rawData(); // dy for input
```

Comment:
Could you give these changes a try please. Make sure to use the numbers from the XML output, since the figures that `ctest` prints include setup and teardown and are thus meaningless. Thanks!